### PR TITLE
Expose bench_local iteration count via commandline

### PR DESCRIPTION
**Pull request by michaelwoerister**
I found this useful for getting less noisy numbers when benchmarking things locally.

r? @Mark-Simulacrum or @rylev

### CODE REVIEW

**Review by Mark-Simulacrum — APPROVED** [review]
Seems good to me - looks like CI is asking for a formatting bump.

File: collector/src/main.rs [hunk 1]
```diff
@@ -117,14 +117,29 @@ fn build_kinds_from_arg(arg: &Option<&str>) -> anyhow::Result<Vec<BuildKind>> {
...
 
+fn iterations_from_arg(arg: Option<&str>) -> anyhow::Result<usize> {
+    if let Some(arg) = arg {
+        if let Ok(iterations) = usize::from_str_radix(arg, 10) {
+            Ok(iterations)
+        } else {
+            anyhow::bail!(
+                "cannot parse iteration count '{}'. Must be a decimal number.",
+                arg
+            );
```
Comment (rylev) [inline collector/src/main.rs:136]: Nit: perhaps just warning and then fallback to 1?

```rust
fn iterations_from_arg(arg: Option<&str>) -> usize {
  arg.and_then(|iterations| {
     match usize::from_str_radix(arg, 10) {
        Ok(i) => Some(i),
        Err(e) => {
            eprintln!("Error parsing iteration count: {}. Falling back to default.", e);
            None
        }
  }).unwrap_or(1)
}
```

**Review by rylev — APPROVED** [review]
Awesome! Only one nit, but we can merge anyway.

**Comment by rylev** [comment]
Going to merge this - @mw let me know if you agree with the comment I had, and if so, I'll create an issue for it.

**Pull request merged**